src/node/interfaces.cpp

```diff
@@ -698,11 +697,9 @@ class ChainImpl : public Chain
     bool checkChainLimits(const CTransactionRef& tx) override
     {
         if (!m_node.mempool) return true;
-        LockPoints lp;
-        CTxMemPoolEntry entry(tx, 0, 0, 0, 0, false, 0, lp);
-        const CTxMemPool::Limits& limits{m_node.mempool->m_limits};
         LOCK(m_node.mempool->cs);
-        return m_node.mempool->CalculateMemPoolAncestors(entry, limits).has_value();
+        std::string err_string;
+        return m_node.mempool->CheckPackageLimits({tx}, m_node.mempool->m_limits, err_string);
```

Comment:
CI failure is due to merge conflict can be fixed with
```suggestion
        return m_node.mempool->CheckPackageLimits({tx}, GetVirtualTransactionSize(*tx), err_string);
```